code/index.html

```diff
@@ -1,30 +1,126 @@
+                    </div>
+                    <div class="radiobg">
+                      <label class="container">Blue Package
+                        <input type="radio" name="radio">
+                        <span class="checkmark"></span>
+                      </label>
+                    </div>
+                    <div class="radiobg">
+                      <label class="container">Green Package
+                        <input type="radio" name="radio">
+                        <span class="checkmark"></span>
+                      </label>
+                    </div>
+                </div>
+                
+                    <select name="gender ">
+                      <option value="Choose ">Choose</option>
+                      <option value="Male ">Male</option>
+                      <option value="Female ">Female</option>
+                      <option value="Intersex/non-binary ">Intersex/non-binary</option>
+                      <option value="Prefer not to respond ">Prefer not to respond</option>
+                   </select>
+                    
+                   <div class="terms">
+                       <input class="termscheck" type="checkbox" id="checkbox">
+                       <label for="checkbox">I agree to these <a href="#">Terms and Condition</a></label>
+                   </div>
+                   <button class="sub-button">Subscribe</button>
+                
+                </form>
+             </div>
+
+            <!-- second animation button that takes you to section 3 -->
+        </div>
+        <a href="#sec-3">
+            <div class="scroll-down halfling " id="scroll-two "></div>
+        </a>
+    </section>
+
+<!-- the FAQ section -->
+    <section class="accordion" id="sec-3">
+
+        <div class="faq">
+            <h1 class="faqtext">Frequently Asked Questions</h1>
+
+        </div>
+        
+        <button class="accordion-button">Question 1</button>
+            <div class="panel">
+            <p>Lorem ipsum...</p>
+            </div>
+
+              <button class="accordion-button">Question 2</button>
+              <div class="panel">
+              <p>Lorem ipsum...</p>
+              </div>
+
+              <button class="accordion-button">Question 3</button>
+              <div class="panel">
+              <p>Lorem ipsum...</p>
+             </div>
+
+             <button class="accordion-button">Question 4</button>
+             <div class="panel">
+             <p>Lorem ipsum...</p>
+            </div>
+
+            <button class="accordion-button">Question 5</button>
+            <div class="panel">
+            <p>Lorem ipsum...</p>
+           </div>
+
+           <button class="accordion-button">Question 6</button>
+           <div class="panel">
+           <p>Lorem ipsum...</p>
+          </div>
+       
+        
+    </section>
+    
+
+    <!--The javascript link! -->
+    <script src="./script.js"></script>
+</body>
+
+</html>
```

Comment:
overall I think your html is easy to follow and understand. it has a good structure! 👍